app/stylesheet/notifications.scss

```diff
@@ -10,80 +10,214 @@
+  border: 1px solid #d1d1d1;
+  -webkit-box-shadow: 0 6px 12px rgb(3 3 3 / 18%);
+  box-shadow: 0 6px 12px rgb(3 3 3 / 18%);
+  overflow-y: auto;
+  position: absolute;
+  right: 0;
+  width: 350px;
+  z-index: 2;
   display: flex;
   flex-direction: column;
   overflow-y: hidden;
-  .panel-group {
-    display: flex;
+  top: 38px;
+  height: calc(100% - 44px);
+  .panel-header {
+    background-color: #fafafa;
+    border-bottom: 1px solid #d1d1d1;
+    position: absolute;
+    width: 100%; 
+    background-image: linear-gradient(to bottom, #fafafa 0%, #ededed 100%);
+    .closeIcon {
+      right: 0px;
+    }
+    .collapsed {
+      left: 0px;
+      padding-left: 5px;
+    }
+    .expanded {
+      left: 0px;
+      padding-left: 5px;
+    }
+    .heading-icon {
+      color: #252525;
+      cursor: pointer;
+      line-height: inherit;
+      position: absolute;
+      background: none;
+    }
+  }
+  
+  .bx--accordion--start .bx--accordion__heading {
+    background-image: linear-gradient(to bottom, #fafafa 0%, #ededed 100%);
+  }
+  
+  .bx--accordion__content {
+    margin: 0px;
+    padding: 0px;
+    flex: 1 1 auto;
+    overflow-y: hidden;
     flex-direction: column;
-    overflow-y: auto;
-    .panel.panel-default {
-      &.expanded {
-        min-height: 175px;
-        flex: 1 1 auto;
-        display: flex;
-        flex-direction: column;
-        overflow-y: hidden;
+    min-height: 0px; 
+    .bx--inline-notification {
+      margin-top: 0px;
+      padding-top: 0px; 
+      margin-bottom: 0px;
+      min-height: 100px;
+      height: auto;
+      .bx--btn--ghost:hover, .bx--btn--ghost:active {
+        background-color: inherit;
       }
-      .panel-collapse {
-        flex: 1 1 auto;
-        overflow-y: auto;
-        display: flex;
-        flex-direction: column;
-        min-height: 0px;
-        .panel-body {
-          padding: 0;
-          flex: 1 1 auto;
-          overflow-y: auto;
+      .bx--inline-notification__text-wrapper {
+        flex-wrap: unset;
+        .notification_time {
+          left: 55px;
+          position: absolute;
+          padding-top: 10px;
+        }
+        .bx--inline-notification__title {
+          margin-bottom: 20px;
+        }
+        .subtitle_div {
+          height: 100%;
+          .subtitle_div_notification_menu {
+            height: 80%;
+          }
+          .subtitle_div_notification_time {
+            height: 20%;
+          }
         }
       }
     }
-  }
-}
-
-.drawer-pf-notification {
-  .dropdown-menu-right.dropdown-menu {
-    li.disabled {
-      cursor: not-allowed;
-      * {
-        pointer-events: none;
+  
+    .read {
+      .bx--inline-notification__title {
+        font-weight: 400;
+      }
+      .bx--inline-notification__subtitle {
+        font-weight: 200;
       }
     }
   }
+     
 }
 
-// display under button
-.drawer-pf.drawer-alt-pf.drawer-pf-notifications {
-  top: 38px;
-  height: calc(100% - 44px);
+
+.Notification-module {
```

Comment:
Minor, but why is this a capital N?  I noticed it in the div above too.

Also noticed sometimes the divs have `-` and sometimes `_`.  Is there a convention/reason for the choice of one or the other?

Reply:
@Fryguy i fixed it and changed it to small N, also fixed _, i was so used to using _ but looks like MIQ ui uses mostly -, changed it now, please review again, thanks.